spec/author_spec.rb

```diff
@@ -0,0 +1,19 @@
+require_relative './models/author'
+require_relative './item'
```

Comment:
To refer to the author file, I think you should go to the previous directory.
Either it should be like:
`require_relative '../models/author'`
`require_relative '../item'`